test/TensorFlowRuntime/tensor.swift

```diff
@@ -178,14 +178,37 @@ TensorTests.testAllBackends("Reduction") {
   #if !TPU
   // 2 x 5
   let x = Tensor<Float>([[1, 2, 3, 4, 5], [1, 2, 3, 4, 5]])
-  expectEqual(ShapedArray(shape: [5], scalars: [2, 4, 6, 8, 10]),
-              x.sum(squeezingAxes: 0).toHost(shape: []).array)
-  expectEqual(ShapedArray(shape: [1, 5], scalars: [2, 4, 6, 8, 10]),
-              x.sum(alongAxes: 0).toHost(shape: []).array)
-  expectEqual(ShapedArray(shape: [5], scalars: [1, 4, 9, 16, 25]),
-              x.product(squeezingAxes: 0).toHost(shape: []).array)
-  expectEqual(ShapedArray(shape: [1, 5], scalars: [1, 4, 9, 16, 25]),
-              x.product(alongAxes: 0).toHost(shape: []).array)
+  expectEqual(Tensor(30), x.sum().toHost(shape: []))
```

Comment:
I wonder if reduction ops still fail on TPU. The `#if !TPU` condition was added months ago (b/111815968).

cc @pschuh @bgogul: I wonder if you have any clue if this test still fails on TPU?
Or if `toHost(shape: ...)` is necessary everywhere?

Reply:
Also, the arguments to `toHost(shape:)` don't seem quite right (it shouldn't always be `[]`).

Reply:
Update: I'll remove the `toHost` calls to force us to re-evaluate this bug.